src/presentation/controllers/ingest-controller.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseController, PostRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { Request, Response } from 'express'
+import { IngestService } from '../../business-logic/services/interfaces/ingest-service'
+
+@RestController('/ingest-operations')
+export class IngestController extends BaseController {
+
+  constructor(private readonly ingestService: IngestService, private readonly httpErrorHandler: HttpErrorHandler) {
+    super()
+  }
+
+  @PostRequest('/rundowns/:rundownName/reingest-data')
```

Comment:
`-data` is a fill word here. `reingest` should be enough to tell what this does.

Reply:
Filler-word has been removed!